tests/migration_test.go

```diff
@@ -901,16 +903,38 @@ var _ = Describe("[rfe_id:393][crit:high[vendor:cnv-qe@redhat.com][level:system]
 				}
 				handler, err := kubecli.NewVirtHandlerClient(virtClient).ForNode(vmi.Status.MigrationState.TargetNode).Pod()
 				Expect(err).ToNot(HaveOccurred())
-				// The port-forwarder tears down after each check, but may be too slow, so use different ports on fast checks
-				for port, _ := range vmi.Status.MigrationState.TargetDirectMigrationNodePorts {
-					func() {
+
+				var wg sync.WaitGroup
+				wg.Add(len(vmi.Status.MigrationState.TargetDirectMigrationNodePorts))
+
+				i := 0
+				errors := make(chan error, len(vmi.Status.MigrationState.TargetDirectMigrationNodePorts))
+				for port, targetPort := range vmi.Status.MigrationState.TargetDirectMigrationNodePorts {
```

Comment:
nit: since `targetPort` is not used anymore, you could replace it with `_` and remove it from the func

Reply:
Will address it.